pauthabielf64/pauthabielf64.rst

```diff
@@ -428,101 +435,66 @@ memory management unit. Platforms may not need to implement all of
+---------
 
 The GOT is a linker generated table of pointers, where each entry is
-created as a result of a GOT-creating relocation present in a
-relocatable object. Many GOT entries will have a dynamic relocation
-that the dynamic linker will resolve at program load time. Once the
-GOT has been relocated it can be re-mapped as read-only (RELRO). If
-the GOT is RELRO then it does not need to be signed, but if the
-platform or link-unit (-z norelro) does not support RELRO for the GOT
-will be writeable for the lifetime of the program. There is scope for
-some or all of the pointers in the GOT to be signed.
-
-If a pointer in the GOT is signed then the dynamic linker must sign the
-pointer at load time, and the code that loads the pointer from the GOT
-must authenticate it using the same signing schema.
-
-The ABI recommends that if RELRO is available the GOT is not
-signed. The standard GOT-generating from AAELF64_ can be used for all
-pointers in the GOT.
-
-PAUTHELF64 describes a default signing schema for GOT entries and
-AUTH variant GOT-generating relocations that can be used to create
-signed pointers in the GOT.
-
-Default signing schema
-^^^^^^^^^^^^^^^^^^^^^^
-
-Signed GOT entries use the ``IA`` key for symbols of type ``STT_FUNC``
-and the ``DA`` key for all other symbol types, with the address of the
-GOT entry as the modifier. The static linker must encode the signing
-schema into the GOT slot. AUTH variant dynamic relocations must be
-used for signed GOT entries.
-
-Example Code to access a signed GOT entry
-
-.. code-block:: asm
-
-  adrp x8, :got_auth: symbol
-  add x8, x8, :got_auth_lo12: symbol
-  ldr x0, [x8]
-  // Authenticate to get unsigned pointer
-  autia x0, x8
-
-In the example the :got_auth: and :got_auth_lo12: operators result in
-AUTH variant GOT generating relocations being used.
-
-Compatibility
-^^^^^^^^^^^^^
-
-If there are AUTH and non-AUTH variant GOT generating relocations to
-the same symbol two GOT entries are required, one signed and one
-unsigned. While not a hard limitation many static linkers only support
-a single GOT entry per symbol. An implementation may choose to fault
-an AUTH and a non-AUTH GOT generating relocation to the same symbol,
-this would require all the GOT-generating relocations to a symbol to
-be signed or unsigned.
+created as a result of a GOT-creating relocation from a relocatable
+object. The GOT is normally split into two subsets, the GOT and the
+PLT GOT. With the PLT GOT made up of code pointers that are only
+accessed by linker generated PLT sequences. As the PLT GOT is not
+accessed directly by code in relocatable objects will be covered in a
+separate section of the document.
+
+The pointers in the GOT may include pointers to code so there is a
+question of whether these pointers should be signed and if so, how
+should they be signed?
+
+The value of many of the pointers in the GOT is not known at static
+link time, these GOT entries will have a dynamic relocation that a
```

Comment:
I think that comma between `time` and `these` should be a period.

Reply:
Ack, will fix as suggested.